src/app/examples/stock-market/components/stock-market-container.component.spec.ts

```diff
@@ -172,5 +173,5 @@ function createState(stockState: StockMarketState) {
     examples: {
       stockMarket: stockState
     }
-  };
+  } as State;
```

Comment:
`as State` vs `:State` what is the tradeoff here ?

Reply:
If you would do `:State` you would have to create your whole State tree, so this would become:


```ts
return {
    examples: {
      stockMarket: stockState,
      todos: {} as TodosState
    },
    auth: {} as AuthState
  };
```

Because we're only interested in the `stockMarket` here, I didn't really like this way. Also if you would add another example state, your tests would fail while the change doesn't impact these tests.

We could create a helper to built the entire state, and we would use this helper to built our entire state in every test?

Reply:
Thank you for the explanation! The solution then seems great because it enables us to ignore changes to other modules and their state which is irrelevant for the particular test. Let's not go with entire state helper so that the tests and their setup is as isolated as possible! 👍